### PR TITLE
Hotfix/compara/86

### PR DESCRIPTION
**Pull request by muffato**
2 changes after testing test.rest, especially one to allow fetching the mouse-strains trees
I can make the same pull-request against master as well

### CODE REVIEW

**Comment by muffato** [comment]
Just a sec, I'll push a unit-test as well

**Comment by nerdstrike** [comment]
Down to the wire eh? Approved.

**Comment by lairdm** [comment]
Waiting for the unit test before these "second eyes" hits merge. Also, @muffato, yes, could you do a pull of this against master too.

File: lib/EnsEMBL/REST/Model/Lookup.pm [hunk 1]
```diff
@@ -148,9 +149,8 @@ sub find_compara_species_sets {
   my $mlsss = $compara_dba->get_MethodLinkSpeciesSetAdaptor->fetch_all_by_method_link_type($method);
   foreach my $mlss (@$mlsss) {
     my $species_set = {};
-    my $species_set_obj = $mlss->species_set_obj();
-    my @species_set_genomes = map { $_->name } @{$species_set_obj->genome_dbs};
-    $species_set->{species_set_group} = $species_set_obj->name;
+    my @species_set_genomes = map { $_->name } @{$mlss->species_set->genome_dbs};
```
Comment (magaliruffier) [inline lib/EnsEMBL/REST/Model/Lookup.pm]: would it be worth keeping the same structure as before and create a species_set variable that is then used in both calls?

**Comment by muffato** [comment]
I've updated the commits, but github doesn't update this page ... Usually it does automatically

**Comment by magaliruffier** [comment]
Have you pushed it? Your branch does not show them either: https://github.com/muffato/ensembl-rest/commits/hotfix/compara/86

**Comment by muffato** [comment]
Yes the commits were there (I've amended them).
I've managed to force a refresh by changing the base branch and back

**Pull request merged**